torch/csrc/jit/passes/onnx/preprocess_for_onnx.cpp

```diff
@@ -97,6 +97,7 @@ static void FuseWithListUnpack(Block* b) {
       case aten::unbind:
       case aten::unsafe_chunk:
       case aten::where:
+      case aten::nonzero_numpy:
```

Comment:
In general, what criteria determines it needs `FuseWithListUnpack` or not?

Reply:
If the list input to `prim::ListUnpack` node is directly an output from a specific parent node, and that output is not used elsewhere.